OONIProbeUnitTests/ResultTest.m

```diff
@@ -32,17 +32,18 @@ - (void)testNoMeasurements {
 }
 
 - (void)testNotUploaded {
-    //One measurement, isEveryMeasurementUploaded false
+    //One measurement, with is_done false, isEveryMeasurementUploaded true
     Result *result = [Result new];
-    Measurement *measurement = [self addMeasurement:result];
-    XCTAssert(![result isEveryMeasurementUploaded]);
+    [self addMeasurement:result];
+    XCTAssert([result isEveryMeasurementUploaded]);
```

Comment:
I do not get exactly why the code change broke the original test. Can you explain?

Reply:
a test who is not done can't be reuploaded

Reply:
So, the function should probably be renamed `testNotDone`?